code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -743,5 +745,11 @@ var/global/list/animal_count = list() //Stores types, and amount of animals of t
 		walk(src,0)
 	return !spaced
 
+/mob/living/simple_animal/say()
+	if(speak_override == TRUE)
+		return ..(pick(speak))
+	else
```

Comment:
You can put return ..() here directly instead of using else

Reply:
if i remove the else than when i set it to 0 and attempt to speak nothing comes out